src/DynamoCoreWpf/Views/Core/ConnectorAnchorView.xaml.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Dynamo.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Threading;
+
+namespace Dynamo.Controls
+{
+    /// <summary>
+    /// Interaction logic for ConnectorAnchorView.xaml
+    /// </summary>
+    public partial class ConnectorAnchorView : UserControl
```

Comment:
some comments on this class and functions here would be appreciated

Reply:
Done.